backend/SensorThingsAPI/src/Org.OpenAPITools/Attributes/ValidateModelStateAttribute.cs

```diff
@@ -0,0 +1,61 @@
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Org.OpenAPITools.Attributes
+{
+    /// <summary>
+    /// Model state validation attribute
+    /// </summary>
+    public class ValidateModelStateAttribute : ActionFilterAttribute
+    {
+        /// <summary>
+        /// Called before the action method is invoked
+        /// </summary>
+        /// <param name="context"></param>
+        public override void OnActionExecuting(ActionExecutingContext context)
+        {
+            // Per https://blog.markvincze.com/how-to-validate-action-parameters-with-dataannotation-attributes/
```

Comment:
Hvem er Per?

Reply:
Good catch. This is auto generated by openAPI so lets just leave it for potential future use.